processing/src/main/java/io/druid/query/groupby/epinephelinae/GroupByQueryEngineV2.java

```diff
@@ -113,6 +113,13 @@ private GroupByQueryEngineV2()
         null
     );
 
+    final boolean allSingleValueDims = query
+        .getDimensions()
+        .stream()
+        .noneMatch(dimension -> {
```

Comment:
I think this would be clearer with allMatch but that might just be personal preference.